### PR TITLE
Skip ILB creation on GCE if neg annotation is present

### PR DESCRIPTION
**Pull request by prameshj**
Internal LoadBalancer resources will be created in a different
controller for these services.

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
 /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by MrHohn — APPROVED** [review]
/lgtm
/approve

File: staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_internal.go [hunk 1]
```diff
@@ -35,7 +35,16 @@ const (
 	allInstances = "ALL"
 )
 
+func usesNEG(service *v1.Service) bool {
+	_, ok := service.GetAnnotations()[NEGAnnotation]
```
Comment (freehan) [inline staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_internal.go]: You probably want to parse the annotation. Probably leave it for TODO for now

**Pull request merged**